enhancements/sig-architecture/30-clusterset-override/README.md

```diff
@@ -327,62 +327,40 @@ So we could finish the migration by four steps, and step 1 and step 2 will be fi
+	// +kubebuilder:validation:Enum=LegacyClusterSetLabel
+	// +kubebuilder:default:=LegacyClusterSetLabel
+	// +required
+	SelectorType SelectorType `json:"selectorType,omitempty"`
 }
 
 type SelectorType string
 
 const (
-    ExclusiveLabel        SelectorType = "ExclusiveLabel"
+	LegacyClusterSetLabel SelectorType = "LegacyClusterSetLabel"
 )
-
-//ExclusiveLabel defines one cluster label
-type ExclusiveLabel struct {
-    //Key is "cluster.open-cluster-management.io/clusterset" by default and can only be cluster.open-cluster-management.io/
-    Key string `json:"key"`
-    //Value can only be empty or the name of the clusterset.
-    Value string `json:"value"`
-}
 ```
 
-- `LabelSelector` will not be included
-- `ExclusiveLabel.Key` must be `cluster.open-cluster-management.io/clusterset` and `ExclusiveLabel.Value` must be `ManagedClusterset Name`    
-- Both `managedclusterset/join` and `managedclusters/label` permission will be supported
-
 2. [Implement in OCM 0.7.0]`multicloud-operators-foundation`, `submariner-addon`, `placement` change the code to integrate with new managedClusterSet api
 
   a. `multicloud-operators-foundation` uses managedClusterSet for resource group purpose. So it should only watch the following managedClusterSets:
-  - `spec.ClusterSelector.SelectorType` is `ExclusiveLabel` and the `ExclusiveLabel.Key` must be `cluster.open-cluster-management.io/clusterset`
-  - `spec.ClusterSelector.SelectorType` is ""
-
-  b. `multicloud-operators-foundation` gives the users `join` permission to a managedClusterSet if the user has "admin" permission to the managedClusterSet. So the `join` permission should be changed with the following rule:
-    ```yaml
-    - apiGroups: ["cluster.open-cluster-management.io"]
-      resources: ["managedclusters/label"]
-      resourceNames: ["cluster.open-cluster-management.io/clusterset:<ManagedClusterSet Name>"]
-      verbs: ["create"]
-    ```
-
-  c. `submariner-addon` uses managedClusterSet group clusters based on the network. And in different managedClusterSet, the clusters should be exclusive. So it should only watch the following managedClusterSet:
-  - `spec.ClusterSelector.SelectorType` is `ExclusiveLabel` and the `ExclusiveLabel.Key` must be `cluster.open-cluster-management.io/clusterset`, the `ExclusiveLabel.Value` must be managedClusterSet name.
-  - `spec.ClusterSelector.SelectorType` is ""
-
-  d. `placement` using new `ClusterSelector` to select target clusters.
+  - `spec.ClusterSelector.SelectorType` is `LegacyClusterSetLabel`
+
+  b. `submariner-addon` uses managedClusterSet group clusters based on the network. And in different managedClusterSet, the clusters should be exclusive. So it should only watch the following managedClusterSet:
+  - `spec.ClusterSelector.SelectorType` is `LegacyClusterSetLabel`
+
+  c. `placement` using new `ClusterSelector` to select target clusters.
 
 3. [Implement in OCM 0.8.0] Update full managedClusterSet api and RBAC
 - Include `LabelSelector`
```

Comment:
I think exclusiveKey is not necessary?

Reply:
> I think exclusiveKey is not necessary?

I am not really understand what you mean, Do you mean we should remove the `ExclusiveLabel` in selectorType ?